### DISCUSSION TITLE
multiselect CDN Do not work((

### DISCUSSION THREAD

**Discussion by RomaBilka** [primefaces/community#1569] [PrimeVue] [answered]
Hello everyone,

https://stackblitz.com/edit/web-platform-1oyh9p?file=index.html

I'm just starting to work with VUE and PrimeVue, and I'm facing some issues.
Options in the select are not filled correctly, objects are simply inserted instead of their values.
as label: { "name": "Rome", "code": "RM" }

I'm using VUE3 CDN.
Does anyone know how to solve this problem?

Also, I need to dynamically load values from the server for the select. How can I do this properly to work with VUE3 CDN and PrimeVue?

`<div id="app">

      <multi-select 
      :options="productType"
      placeholder="Select Product Type"
      optionLabel="name"
      optionValue="code"
      class="w-full " />

    </div>

    <script>
      const { createApp, ref } = Vue;

      const app = createApp({
        setup() {
          const productType = ref([
            { name: 'New York', code: 'NY' },
            { name: 'Rome', code: 'RM' },
            { name: 'London', code: 'LDN' },
            { name: 'Istanbul', code: 'IST' },
            { name: 'Paris', code: 'PRS' }
        ])
          const date = ref();
          return {
            date,
            productType,
          };
        },
      });

      app.use(primevue.config.default);
      app.component('multi-select', primevue.multiselect);
      app.mount('#app');
    </script>`

**Comment by r0drigocarvalho** [comment, marked as answer]
Hi, i see a solution for de bug here: https://github.com/primefaces/primevue/issues/5151

Solution for your code:

`<multi-select 
      :options="productType"
      placeholder="Select Product Type"
      option-label="name"
      option-value="code"
      class="w-full"></multi-select>`

**Reply by RomaBilka** [reply]
thank you bro :))
